include/belle.hh

```diff
@@ -1348,7 +1348,7 @@ private:
             {
               return static_cast<int>(e);
             }
-            catch (std::exception const& e)
+            catch (std::exception const&)
```

Comment:
In the future, I'd like to add a logging interface which will end up using this variable. To suppress the warning in the mean time, the following could be used:
```cpp
catch (std::exception const& e)
{
  boost::ignore_unused(e);

  return 500;
}
```

Reply:
I dont see why you prefer adding boost::ignore_unused (to remove it later, then you add logging interface) over just adding the " e" part then you add the logging interface?!